src/utils.ts

```diff
@@ -8,6 +8,24 @@
 import { isArray } from '@salesforce/ts-types';
 import { capitalCase } from 'change-case';
 import { JsonObject } from '@salesforce/sf-plugins-core';
+import { Messages } from '@salesforce/core';
+import { AuthInfo } from '@salesforce/core';
+
+Messages.importMessagesDirectory(__dirname);
+const messages = Messages.loadMessages('@salesforce/plugin-env', 'util');
+
+export type OrgSideEffects = {
```

Comment:
This type seems to be duplicated here and in ./src/shared/util.ts. Can we just have it one place?